dev/MRTCore/mrt/Core/src/MRM.vcxproj

```diff
@@ -124,6 +126,10 @@
            than they would otherwise be if the CRT, runtime, and STL were all statically linked in. -->
       <IgnoreSpecificDefaultLibraries>%(IgnoreSpecificDefaultLibraries);libucrt.lib</IgnoreSpecificDefaultLibraries>
       <AdditionalOptions>%(AdditionalOptions) /defaultlib:ucrt.lib</AdditionalOptions>
+      <!-- Configure the release build binary to be as required by internal API scanning tools.
+           Note: /opt:icf and /opt:ref achieve the same result as setting EnableCOMDATFolding and OptimizeReferences to
+           true, which is done below for all build configurations, so we exclude them here. -->
+      <AdditionalOptions>%(AdditionalOptions) /debug /debugtype:cv,fixup /incremental:no</AdditionalOptions>
```

Comment:
Pending: per John's comment, I'll replace all direct option setting in this change with individual linker/build settings (like we have for `EnableCOMDATFolding` and `OptimizeReferences` currently).

Reply:
addressed